### DISCUSSION TITLE
Unable to send question set for approval and request changes

### DISCUSSION THREAD

**Discussion by agamdahiya** [Sunbird-coKreat/Community#100] [General Discussions]
Hi team.

I am unable to send the question set for approval and request in VDN .  I am getting this error
 Request URL:
https://vdn.diksha.gov.in/action/question/v4/system/update/do_4314011721539371008142
Request Method:
PATCH
Status Code:
400 Bad Request

{
    "id": "api.question.system.update",
    "ver": "3.0",
    "ts": "2024-03-21T13:04:20ZZ",
    "params": {
        "resmsgid": "ca164415-0bd8-4889-8254-648570b080f3",
        "msgid": null,
        "err": "CLIENT_ERROR",
        "status": "failed",
        "errmsg": "Invalid request"
    },
    "responseCode": "CLIENT_ERROR",
    "result": {
        "messages": [
            "Invalid Props are : [reviewerQualityChecks]"
        ]
    }
}

 Attaching the screenshot for your reference.
![image](https://github.com/Sunbird-coKreat/Community/assets/127106035/b23c2cfc-c235-44f5-a64d-c392ce79c4c9)

![image](https://github.com/Sunbird-coKreat/Community/assets/127106035/acdcbfdf-d6d7-4e1c-b7d0-82b86fd5d1e9)

The **submit for approval and request change** button are not working
![image](https://github.com/Sunbird-coKreat/Community/assets/127106035/fcd26eb9-a800-46db-82d9-3ec28b51ea16)

Requesting you to help us asap.

**Comment by vinukumar-vs** [comment]
@agamdahiya It's clearly saying the invalidProperty(`reviewerQualityChecks`) is sending in the request payload of the question-set create.

Verify this property is add in the question-set schema and redeploy the coKreat portal and assessment service along with Content-service.

[@sajeshkayyath](https://github.com/sajeshkayyath) [@PoojaJManjooran](https://github.com/PoojaJManjooran)

**Comment by agamdahiya** [comment]
@vinukumar-vs 

Thanks for the solution, it is working fine after the implementation.
